.github/workflows/tests.yml

```diff
@@ -0,0 +1,39 @@
+name: run_tests
+
+on: [push, pull_request]
+
+jobs:
```

Comment:
your work here resolves issue on this project: https://github.com/moremoban/pypi-mobans/issues/148

which meant: all projects using pypi-mobans would benefit from your work, including pyexcel family projects.

Reply:
cool! :)

and yes, it's as simple as that:
```
matrix:
    python-version: [3.6, 3.7, 3.8, 3.9]
    os: [macOs-latest, ubuntu-latest, windows-latest]
```